website/blog/2024/07-05-launching-crawlee-python/index.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: launching-crawlee-python
+title: 'Announcing Crawlee Python: Now you can use Python to build reliable web crawlers'
```

Comment:
```suggestion
title: 'Announcing Crawlee for Python: Now you can use Python to build reliable web crawlers'
```

also the title is huge, it doesn't look very nice

<img width="1020" alt="image" src="https://github.com/apify/crawlee/assets/615580/d367e479-44b2-4933-9a66-3c201a76b626">

i would personally keep just the `Announcing Crawlee Python` and put the second part into the content